pkg/daemon/ceph/client/filesystem.go

```diff
@@ -174,6 +175,13 @@ func IsMultiFSEnabled() bool {
 	}
 	return false
 }
+func SetPools(context *clusterd.Context, cephVersion cephver.CephVersion, clusterName, fsName string, dataPool []*model.Pool, metdataPool *model.Pool) error {
+	args_metadata := []string{"osd", "pool", "set", fsName + "-metadata", "size", strconv.Itoa(int(metdataPool.ReplicatedConfig.Size))}
+	NewCephCommand(context, clusterName, args_metadata).Run()
```

Comment:
Instead of calling `NewCephCommand()`, call `SetPoolProperty()` in the `pkg/daemon/ceph/client` package to change this replication.